x/nft/proto/buf.yaml

```diff
@@ -0,0 +1,21 @@
+version: v1
+name: buf.build/mods/nft
```

Comment:
I cannot push this because `buf.build/cosmos/cosmos-sdk` already defines it.
Looking how to force the push with the use of draft commits.

Reply:
Failure: failed to push module, pushing a module with a dependency pinned to a draft commit is not allowed, dependency "buf.build/cosmos/cosmos-sdk" is pinning to a draft commit "eb123a948f1c4f12881e020940fe007b" 🤡